src/plugins.js

```diff
@@ -9,12 +9,31 @@ import requestAuthPlugin from './plugins/requestAuthPlugin';
  * @module ServerPlugins
  */
 
+export function setCsrfCookies(request, reply) {
+	const csrfHeader = (request.response.headers || {})['x-csrf-jwt'];
+	if (csrfHeader) {
+		reply.state('x-csrf-jwt-header', csrfHeader);
+	}
+	return reply.continue();
+}
+
 export function getCsrfPlugin(secret) {
 	const register = (server, options, next) => {
-		server.state('x-csrf-jwt', {
+		const cookieOptions = {
+			path: '/',
 			isSecure: process.env.NODE_ENV === 'production',
-		});
-		return CsrfPlugin.register(server, options, next);
+		};
+		server.state('x-csrf-jwt', {
+			...cookieOptions,
+			isHttpOnly: true,
+		});  // set by plugin
+		server.state('x-csrf-jwt-header', {
```

Comment:
OK, so as I understand `server.state`, it essentially sets cookies.

And as I understand this, we are setting two cookies with no value, and then `x-csrf-jwt-header` gets set to the header `onPreResponse`.

What are we doing with `x-csrf-jwt`?

Possible I'm missing some other code elsewhere.